### DISCUSSION TITLE
[question] How to apply git patch?

### DISCUSSION THREAD

**Discussion by uilianries** [gitpython-developers/GitPython#1136] [Q&A] [answered]
Hi!

I would like to apply a patch file as usual:

    git apply 0001-foobar.patch

How could I execute such operation using GitPython?

**Comment by Byron** [comment, marked as answer]
It will probably be easiest to [use git directly](https://gitpython.readthedocs.io/en/stable/tutorial.html#using-git-directly).

In case there are follow up questions or concerns, please feel free to posting to this issue.
Good luck!

**Comment by uilianries** [comment]
Thank you!
